### ISSUE TITLE
[Bug]: Installing NGINX with non ascii chars in username fails

### ISSUE THREAD

**Issue by geisi** [beyondcode/herd-community#508]
### Operating system version

Windows 10

### System architecture

Intel (x86)

### Herd Version

1.0.0

### Bug description

When Installing Herd on Windows 10 using a Windows User with MultiByte Characters in its Home Folder Herd is not able to install NGINX properly. I think this is not specific to Windows 10 and a general problem.

### Steps to reproduce

1. Create a Windows User with Multibyte Characters in its Home Folder. E.g. Tim Geisendörfer
2. Install Herd

### Relevant log output

```shell
NGINX Log:
[emerg] 9484#1396: CreateFile() "C:\Users\Tim Geisend�rfer\.config\herd\config\nginx\nginx.conf" failed (1113: No mapping for the Unicode character exists in the target multi-byte code page)
```

**Comment by felixworen** [comment]
Same error for me with an é in my user name

**Comment by mpociot** [comment]
Fixed in version 1.0.1 
https://herd.laravel.com/changelog/windows#101
